core/nuxeo-page-provider.js

```diff
@@ -428,25 +428,22 @@ import './nuxeo-resource.js';
         params.query = this.query;
       }
 
-      if (this.params.queryParams) {
-        // these should've simply been passed as an array instead
-        params.queryParams = this.params.queryParams;
-      } else if (Array.isArray(this.params)) {
-        params.queryParams = this.params;
-      } else {
-        const namedParams = {};
-        Object.keys(this.params).forEach((key) => {
-          let value = this.params[key];
-          if (value != null) {
-            if (Array.isArray(value)) {
-              value = JSON.stringify(value.map((item) => (item['entity-type'] ? item.uid || item.id : item)));
-            } else if (typeof value !== 'string') {
-              value = value['entity-type'] ? value.uid || value.id : JSON.stringify(value);
-            }
-            namedParams[key] = value;
+      const { queryParams } = this.params;
+      const namedParams = {};
+      Object.keys(this.params).forEach((key) => {
+        let value = this.params[key];
+        if (value != null) {
+          if (Array.isArray(value)) {
+            value = JSON.stringify(value.map((item) => (item['entity-type'] ? item.uid || item.id : item)));
+          } else if (typeof value !== 'string') {
+            value = value['entity-type'] ? value.uid || value.id : JSON.stringify(value);
           }
-        }, this);
-        params.namedParameters = namedParams;
+          namedParams[key] = value;
```

Comment:
This means that if we have `queryParams` on `this.params`, we will end up with `queryParams` inside both `params`and `params.namedParameters`, right? So it will be duplicated.